### PR TITLE
Added Emotes and Sounds to Corgis and Cats. Moved Animal Sounds to Creature Folder. 

### PR DESCRIPTION
**Pull request by Tails2091**
Added bark and growl emote to corgis, as well as two barking sounds and a yelp.
For cats I've added meow, hiss, and purr for emotes and a meow sound.
Video Example of these sounds: [VIDEO](https://drive.google.com/file/d/1d4cFVh1E4MQbWilusMuTajP8saICGzc6/view?usp=sharing)

I also moved caw.ogg, hoot.ogg, mousesqueak.ogg, and nymphchirp.ogg to the creatures folder as the only sound using this directory was bee.ogg.

:cl:
add: New emotes to Cats and Corgis.
tweak: Change directories for some creature sounds to be in the creature sound folder.
soundadd: Two bark sounds, a yelp, and a meow.
/:cl:

Sources:
Yelp: https://freesound.org/people/TobiasKosmos/sounds/163280/ (CC BY 3.0) credit given in license.txt
Both barks: https://freesound.org/people/apolloaiello/sounds/276267/ (CC0 1.0)
Meow: heavily edited version of both https://freesound.org/people/KaiAndersen/sounds/454805/ https://freesound.org/people/skymary/sounds/412016/ (CC0 1.0)

### CODE REVIEW

**Comment by angelomerp** [comment]
heey, that's actually pretty good, I like it


*dramatic moment:* OH NOOOOOO, POOR DOGGO!!

File: sound/creatures/license.txt [hunk 1]
```diff
@@ -0,0 +1,2 @@
+dog_yelp.ogg is DogYelp.wav, by TobiasKosmos (https://freesound.org/people/TobiasKosmos/sounds/163280/). It has been licensed under CC-BY 3.0 license.
+It has been cropped, EQ'd, and compressed to .ogg for in-game use.
```
Comment (Fox-McCloud) [inline sound/creatures/license.txt]: Let's not add an entire text file fo a single sound.

Comment (Tails2091) [reply sound/creatures/license.txt]: Where should I store license information?

**Comment by Tails2091** [comment]
I've merged both the license.txt files into Attributions.txt.

**Pull request merged**